assignments/lambda-classes.js

```diff
@@ -1 +1,194 @@
 // CODE here for your Lambda Classes
+//Person Object Creator
+class Person{
+    constructor(attributes){
+        this.name = attributes.name;
+        this.age = attributes.age;
+        this.location = attributes.location;
+        this.gender = attributes.gender;
```

Comment:
Here is an example where you could use object destructuring straight from your parameters:
```javascript
constructor({ name, age, location, gender }) {
    this.name = name;
    this.age = age;
    this.location = location;
    this.gender = gender;
}
```